var/spack/repos/viz/repo.yaml

```diff
@@ -0,0 +1,2 @@
+repo:
+  namespace: viz
```

Comment:
this repo won't be enabled by default, could you enable it in [etc/spack/defaults/repos.yaml](https://github.com/BlueBrain/spack/blob/develop/etc/spack/defaults/repos.yaml) ([docs](https://spack.readthedocs.io/en/latest/repositories.html?highlight=repo#repos-yaml) ) ?

Reply:
Done